src/tensor/wrappers.jl

```diff
@@ -0,0 +1,315 @@
+using CUDAdrv: CuDefaultStream, CuStream, CuStream_t
+
+function cudaDataType(T::DataType)
```

Comment:
This would make sense to move into CUDAapi instead, as it's probably used by CUBLAS too.

Reply:
We aren't using it in CUBLAS yet because we don't have tests or anything for mixed precision gemmed